src/contentful/utils.ts

```diff
@@ -42,16 +61,22 @@ export const getContentfulRequest = (type, contentQuery) => {
   };
 };
 
-const hasValidationErrors = (payload) => payload?.errors;
+const hasValidationErrors = (payload?: Record<string, unknown>) => payload?.errors;
```

Comment:
missing return type. why doesn't `tsc` complain about it?

Reply:
Because return type is sorta optional?